### DISCUSSION TITLE
Unable to properly run Chia GUI after power failure

### DISCUSSION THREAD

**Discussion by Sir-WilliamG** [Chia-Network/chia-blockchain#6695] [Support]
I've been running the GUI without issue for a while now.  Yesterday, we had a power failure while Chia was running.  Ever since, when I run the GUI, it stalls before I can even select the key.  I tried uninstalling, and re-installing, but that didn't fix the issue.  The screenshot included is as far as I get.  Any help would be appreciated...

![Chia Problem](https://user-images.githubusercontent.com/85782435/121769114-6c88cd80-cb27-11eb-8f6d-aab15a8cb886.png)

**Comment by loppefaaret** [comment]
It's likely a corrupted database, first try resetting the wallet db with these instrucitons:
Close down the Chia app, and open up a PowerShell window, and execute the below command:
```
del ~\.chia\mainnet\wallet\db\block*
```
Then try opening up the Chia app again

**Reply by Sir-WilliamG** [reply]
Still no joy.  Any other ideas I can try?

**Reply by loppefaaret** [reply]
Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old
cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\
.\chia.exe init
mv ~\.chia\old\db ~\.chia\mainnet\
mv ~\.chia\old\wallet ~\.chia\mainnet\
```
this will do a complete reset of the configuration, but keep databses

**Reply by Sir-WilliamG** [reply]
That did the trick.  Thank you very much for your help!

**Comment by BrandtH22** [comment]
@Sir-WilliamG , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
